topologies/binding/binding.go

```diff
@@ -162,9 +162,15 @@ func (b *staticBind) DialIxNetwork(ctx context.Context, ate *binding.ATE) (*bind
 	if err != nil {
 		return nil, err
 	}
-	ixs, err := ixw.IxNetwork().NewSession(ctx, ate.Name)
-	if err != nil {
-		return nil, err
+	var ixs *ixweb.Session
+	var ixs_err error
+	if dialer.SessionId > 0 {
+		ixs, ixs_err = ixw.IxNetwork().FetchSession(ctx, int(dialer.SessionId))
+	} else {
+		ixs, ixs_err = ixw.IxNetwork().NewSession(ctx, ate.Name)
```

Comment:
Per @greg-dennis's comment, let's also delete the new session in staticBind's Release() if we created it.

https://github.com/openconfig/featureprofiles/issues/123#issuecomment-1091045073

Reply:
At the moment, Ondatra isn't calling DialIxNetwork multiple times, but for the sake of correctness, we should remember all the sessions that were created by this bind and delete them on Release().

For this to work, we may need to split the code that creates the ixw client (perhaps move that to dialer.newIxWebClient) and the code that creates new session or fetches existing ones, so that Release() has a ixw client it can use to delete a session.

Reply:
Alternatively, in the absence of a SessionId, `Reserve` can create a new session and store the session ID. Then DialIxNetwork unconditionally fetches the session ID (be it one that was set from the outset, or created by Reserve). Then Release deletes the session, if one was created.

Reply:
Let's go for @greg-dennis's suggestion. I think this means that:

 1. staticBind should maintain its own singleton instance of ixweb client to be obtained from dialer.newIxWebClient. This will be used by Reserve() and Release().
 2. In Reserve(), if ATE session ID is missing, call ixweb client NewSession. Or use FetchSession. Remember this as a singleton instance of *binding.IxSession.
 3. DialIxNetwork always returns the singleton instance of the *binding.IxSession.
 4. In Release(), it only calls DeleteSession if the session was created by Reserve().

Reply:
Thanks @liulk and @greg-dennis for the suggestion. I'm working on those changes now and will try to push commit later today.

Reply:
Hi @liulk and @greg-dennis, I added the logic to use singleton instance to IxWeb and IxSession. Those would be initiated in Reserve() function and would be used in Release() function to delete new session if created. Could you please help check if it matches you expectation?